core-bundle/contao/dca/tl_templates.php

```diff
@@ -72,13 +70,15 @@
 				'href'                => 'act=paste&amp;mode=copy',
 				'icon'                => 'copy.svg',
 				'attributes'          => 'data-action="contao--scroll-offset#store"',
+				'button_callback'     => array('tl_templates', 'copy')
```

Comment:
Maybe add a reminder that this is a workaround?

```suggestion
				// TODO: remove this again once #7854 has been fixed
				'button_callback'     => array('tl_templates', 'copy')
```

Reply:
Good point 👍